kafka-rest/src/main/java/io/confluent/kafkarest/entities/ConsumerGroupLag.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.entities;
+
+import com.google.auto.value.AutoValue;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+@AutoValue
+public abstract class ConsumerGroupLag {
+
+  ConsumerGroupLag() {
+  }
+
+  public abstract Long getMaxLag();
+
+  public abstract Long getTotalLag();
+
+  public abstract String getClusterId();
+
+  public abstract String getConsumerGroupId();
+
+  public abstract String getMaxLagClientId();
+
+  public abstract String getMaxLagConsumerId();
+
+  public abstract Optional<String> getMaxLagInstanceId();
+
+  public abstract String getMaxLagTopicName();
+
+  public abstract Integer getMaxLagPartitionId();
+
+  public static Builder builder() {
+    return new AutoValue_ConsumerGroupLag.Builder();
+  }
+
+  @AutoValue
+  abstract static class Offset {
+
+    abstract String getTopicName();
+
+    abstract String getConsumerId();
+
+    abstract String getClientId();
+
+    abstract Optional<String> getInstanceId();
+
+    abstract int getPartitionId();
+
+    abstract long getCurrentOffset();
+
+    abstract long getEndOffset();
+
+    private long getLag() {
+      return getEndOffset() - getCurrentOffset();
+    }
+
+    private static Builder builder() {
+      return new AutoValue_ConsumerGroupLag_Offset.Builder();
+    }
+
+    @AutoValue.Builder
+    abstract static class Builder {
+
+      abstract Builder setTopicName(String topicName);
+
+      abstract Builder setConsumerId(String consumerId);
+
+      abstract Builder setClientId(String clientId);
+
+      abstract Builder setInstanceId(Optional<String> instanceId);
+
+      abstract Builder setPartitionId(int partitionId);
+
+      abstract Builder setCurrentOffset(long currentOffset);
+
+      abstract Builder setEndOffset(long endOffset);
+
+      abstract Offset build();
+    }
+  }
+
+  @AutoValue.Builder
+  public abstract static class Builder {
+
+    private Long maxLag;
+    private long totalLag = 0;
+
+    private final Set<String> consumers = new HashSet<>();
```

Comment:
What do you use `consumers` for? You add stuff in here, but you never query. Please, remove if you are not using it for anything. Same for any other piece of information that's not used anywhere in REST Proxy.